### DISCUSSION TITLE
ICESat-2 Data request

### DISCUSSION THREAD

**Discussion by adriankreiner** [microsoft/PlanetaryComputer#221] [Data request]
Dear Microsoft Planetary Computer,
 
I hope this message finds you well. 
First, thanks a lot for all your effort and this amazing technology!!!
I am writing to request the addition of the ICESat-2 dataset to the Planetary Computer catalogue. The ICESat-2 (Ice, Cloud, and Land Elevation Satellite-2) is a NASA satellite mission specifically designed to measure the height of Earth's surface, particularly the cryosphere and polar regions. It provides valuable data that would significantly enhance the capabilities of the GEE and enable its users to conduct more diverse and detailed studies.
 
The ICESat-2 dataset, specifically the ATL08 (Land and Vegetation Height) and ATL13 (Inland Water Surface Height) products, are particularly useful for generating lake bathymetry, among many other applications. These products offer a remarkable opportunity to study and monitor changes in Earth's freshwater bodies, contributing significantly to research in hydrology, climate science, and more.
 
Beyond lake bathymetry, there are numerous other potential applications for ICESat-2 data in GEE, including but not limited to monitoring glacier and ice sheet elevations, assessing forest canopy heights, and tracking urban development.
 
By incorporating ICESat-2 data into the Planetary Computer, I believe it would significantly expand the breadth and depth of environmental analyses that could be conducted using your platform.
 
Thank you for considering this request. I appreciate the work your team does in making high-quality Earth observation data accessible and usable for researchers around the world.
 
Looking forward to the possibility of seeing ICESat-2 data in the Planetary Computer portfolio.
 
Best regards,
 
Adrian Kreiner

**Comment by TomAugspurger** [comment]
Thanks for reaching out.

Right now, we're working through a backlog of datasets, so we can make any promises on when we could get to this. But if / when we get to ICESat-2 I'll post an update here.
